yellowbrick/classifier/confusion_matrix.py

```diff
@@ -80,6 +84,7 @@ def __init__(self, model, ax=None, classes=None, **kwargs):
         self.cmap.set_under(color='w')
         self.cmap.set_over(color='#2a7d4f')
         self.edgecolors=[] #used to draw diagonal line for predicted class = true class
+        self.label_mapping= label_mapping
```

Comment:
I made a code change where the user can pass a dictionary mapping or a trained `LabelEncoder`